packages/wallet/src/redux/sagas/adjudicator-watcher.ts

```diff
@@ -52,6 +52,15 @@ function* dispatchEventAction(event: AdjudicatorEvent) {
         })
       );
       break;
+    case AdjudicatorEventType.ChallengeCleared:
+      const newTurnNumRecord = event.eventArgs[1].toNumber();
```

Comment:
You should be able to do `event.args.newTurnNumRecord`

Reply:
`args` is not a property of `event` though.